Specs/Number/English/NumberModel.json

```diff
@@ -2838,5 +2838,69 @@
         "End": 16
       }
     ]
+  },
+  {
+    "Input": " half million",
+    "NotSupported": "javascript, java, python",
+    "Results": [
+      {
+        "Text": "half million",
+        "TypeName": "number",
+        "Resolution": {
+          "subtype": "fraction",
+          "value": "500000"
+        },
+        "Start": 1,
+        "End": 12
+      }
+    ]
+  },
+  {
+    "Input": " quarter million",
```

Comment:
It also works without the first whitespace, right? :)

Reply:
yes of course